### DISCUSSION TITLE
Nemo: Managing pairs of Web pages and folders

### DISCUSSION THREAD

**Discussion by Thontin** [linuxmint/discussions#436] [Ideas]
It would help in managing web pages and their associated folders if Nemo  could associate each .html files with the relevant (_files) folder. At present if I want to manage a web page ; to rename, copy, cut, paste or drag&drop, I select say abcd.html file, apply the action; then have to find the  abcd_files folder and apply the same action. it would be great if I could select either a .html or a _files and Nemo would select the other part of the pair; for immediate  rename, copy, cut, paste or drag&drop action.

**Comment by Jopp-gh** [comment]
Just select multiple files and press the rename shortcut. You can find your rename button under: Nemo's menu bar > help > shortcuts.
Maybe I misunderstood your request.

Btw, HTML files are always linked to their resource folders. In wondering why you want to rename HTML files and folders.
